### DISCUSSION TITLE
Deposit to another address is sent it out o other address

### DISCUSSION THREAD

**Discussion by FarnazKara** [Chia-Network/chia-blockchain#18848] [Support]
On November 3rd, I sent 2 XCH (Chia coins) to the address provided by the Crypto.com platform. However, the transaction has not yet been deposited into my Crypto.com account.

My wallet UI shows the transaction as successful, and the recipient address initially matched the one provided by Crypto.com (ending in "z4pt").

However, when I looked up the transaction on a Chia explorer using the confirmed height (6,161,096), the recipient address displayed there ends in "sud2," which differs from the address I confirmed in my wallet.

I'm concerned about this discrepancy. Can you please help me understand why the recipient address appears different in the explorer? Additionally, I'd appreciate clarification on how to successfully deposit XCH from my wallet to Crypto.com.

Finally, is it possible to recover the 2 XCH coins if the transaction ends up being sent to the incorrect address?

Thank you for your assistance.

**Comment by BrandtH22** [comment]
duplicate
